### PR TITLE
iza.ne.jp #24067

### PR DESCRIPTION
**Pull request by slavaleleka**
https://github.com/AdguardTeam/AdguardFilters/issues/24067

only for web version, for android won't fix

### CODE REVIEW

File: JapaneseFilter/sections/css_extended.txt [hunk 1]
```diff
@@ -27,5 +27,6 @@ mindhack2ch.com##.article-body > .rss:has(> script)
 mindhack2ch.com##.article-post-bottom > .rss-article:has(> .rss-article-item)
 !
 momon-ga.com##.content > .section-box[-ext-has='a[target="_blank"]']
+iza.ne.jp#$#.sideFixedWrapper { position: relative!important; }
```
Comment (BlazDT) [inline JapaneseFilter/sections/css_extended.txt]: Not css_extended. It is general_extensions.
I also don't see a difference with this rule.

File: JapaneseFilter/sections/specific.txt [hunk 2]
```diff
@@ -99,6 +99,15 @@ weblio.jp##.adBannerHdWrp
 weblio.jp##.adSmpBnrFtBtSp
 weblio.jp##.adBkOverlayClr
 new-akiba.com###bannerB2
+iza.ne.jp##.header-link > ul > li > a[href="http://www.sankei-digital.co.jp/"]
+||prt.iza.ne.jp/common/css/v2/images/bg_header_003.png^$domain=iza.ne.jp
```
Comment (BlazDT) [inline JapaneseFilter/sections/specific.txt]: Request not reproduced - also $domain is unnecessary.
Same for other url block rule.

File: JapaneseFilter/sections/specific.txt [hunk 3]
```diff
@@ -99,6 +99,15 @@ weblio.jp##.adBannerHdWrp
...
+||prt.iza.ne.jp/common/images/v1/basic/banner_sankeiPhoto.png^$domain=iza.ne.jp
+iza.ne.jp##article.article-detail-section > .tool > .credit
+iza.ne.jp##.sideFixedWrapper > div[style="width:300px;height:600px;"]
+iza.ne.jp##section.module-pickup-topics
```
Comment (BlazDT) [inline JapaneseFilter/sections/specific.txt]: Breaks good content.

File: JapaneseFilter/sections/specific.txt [hunk 4]
```diff
@@ -99,6 +99,15 @@ weblio.jp##.adBannerHdWrp
...
+iza.ne.jp##aside.ad-special
+iza.ne.jp##aside.ad-trend
+iza.ne.jp##aside.ad-tokudane
```
Comment (BlazDT) [inline JapaneseFilter/sections/specific.txt]: All those aside.ad rules can be done with this rule:
iza.ne.jp##aside[class^="ad-"]

File: SpywareFilter/sections/specific.txt [hunk 5]
```diff
@@ -22,6 +22,7 @@
 ||track.tagesanzeiger.ch/cre*/static/tracking^
 ||injapan.com/rest/*/track?*&callback=
 ||youlead.pl/track?
+||smp.sankei.co.jp/js/analytics/skd.Analysis.js^$domain=iza.ne.jp
```
Comment (BlazDT) [inline SpywareFilter/sections/specific.txt]: Change to:
`||smp.sankei.co.jp/js/analytics^`

**Review by BlazDT — CHANGES_REQUESTED** [review]

**Comment by slavaleleka** [comment]
@BlazDT 
without `iza.ne.jp#$#.sideFixedWrapper { position: relative!important; }`
`div#SecondaryContents` sticks to the bottom after scrolling in desktop browser

File: JapaneseFilter/sections/general_extensions.txt [hunk 6]
```diff
@@ -19,6 +19,8 @@ weblio.jp#$#body[onload=""] { padding-bottom: 0!important; }
 ! https://github.com/AdguardTeam/AdguardFilters/issues/12280
 ogeji.hatenablog.com#$#.sleeping-ads {position: absolute!important; left: -3000px!important; }
 !
+iza.ne.jp#$#.sideFixedWrapper { position: relative!important; }
```
Comment (Alex-302) [inline JapaneseFilter/sections/general_extensions.txt]: add a comment with URL of an issue, as above

Comment (Alex-302) [reply JapaneseFilter/sections/general_extensions.txt]: or description of the rule

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**